lib/proxy.js

```diff
@@ -19,18 +20,25 @@ module.exports = function (app, options) {
 	}
 
 	app.use(apiPath, middleware(options.proxy));
-	app.use('/socket.io/', middleware(options.proxy + '/socket.io/'));
+
+	app.use('/socket.io/', middleware(baseDomain(options.proxy) + 'socket.io/'));
```

Comment:
The `/socket.io` should be configurable since you can change it when setting up Socket.io